docs/content.zh/docs/deployment/elastic_scaling.md

```diff
@@ -141,7 +141,7 @@ Adaptive 调度器可以通过[所有在名字包含 `adaptive-scheduler` 的配
 
 ### 局限性
 
-- **只支持流式 Job**：Adaptive 调度器的第一个版本仅支持流式 Job。当提交的是一个批处理 Job 时，我们会自动换回默认调度器。
+- **只支持流式 Job**：Adaptive 调度器的仅支持流式 Job。当提交的是一个批处理 Job 时，我们会自动换回 Adaptive Batch Scheduler。
```

Comment:
```suggestion
- **只支持流式 Job**：Adaptive 调度器仅支持流式 Job。当提交的是一个批处理 Job 时，我们会自动换回 Adaptive Batch Scheduler。
```

Reply:
Thanks @lsyldliu for the quick review!

Good catch, it's my mistake. Updated.